client/scss/components/_image_gallery.scss

```diff
@@ -30,3 +32,159 @@
     }
   }
 }
+
+// javascript slider styles
+@include slider('image-gallery__');
+
+.image-gallery__slider {
+  overflow: hidden;
+  position: relative;
+  z-index: 0;
+  margin: -160px map-get($container-padding, 'small') * -1 0;
+
+  @include respond-to('medium') {
+    margin-left: map-get($container-padding, 'medium') * -1;
+    margin-right: map-get($container-padding, 'medium') * -1;
+  }
+
+  @include respond-to('large') {
+    margin-left: map-get($container-padding, 'large') * -1;
+    margin-right: map-get($container-padding, 'large') * -1;
+  }
+
+  &:before {
+    height: 60px;
+    display: block;
+    width: 100%;
+    content: '';
+    background: color('white');
+    margin-bottom: -5px;
+    clip-path: polygon(0% 75%, 0% 100%, 100% 100%, 100% 60%, 92% 85%, 70% 67%, 57% 90%, 45% 59%, 20% 72%);
+  }
+}
+
+.image-gallery__slider-inner {
+  overflow: visible;
+  background: color('white');
+  padding: 100px map-get($container-padding, 'small') 0;
+
+  @include respond-to('medium') {
+    padding-left: map-get($container-padding, 'medium');
+    padding-right: map-get($container-padding, 'medium');
+  }
+
+  @include respond-to('large') {
+    padding-left: map-get($container-padding, 'large');
+    padding-right: map-get($container-padding, 'large');
+  }
+}
+
+.image-gallery__slides-container { // these width and left calculations are related to the gallery appearing in the article template
+  margin: 0;
+  position: relative;
+  width: calc(100vw - #{map-get($container-padding, 'small')});
+
+  @include respond-to('medium') {
+    width: calc(100vw - #{map-get($container-padding, 'medium')} - 12.5%);
+    margin-left: 12.5%;
+  }
+
+  @include respond-to('large') {
+    width: calc(100vw - #{map-get($container-padding, 'large')});
+    margin-left: 0;
+  }
+
+  @include respond-to('xlarge') {
+    width: $container-width-max - (map-get($container-padding, 'large') * 2);
+    margin-left: 50%;
+    left: (($container-width-max - (map-get($container-padding, 'large') * 2)) / 2) * (-5/6);
+  }
+}
+
+.image-gallery__slide-item {
+  opacity: 0.2;
+  transition: opacity 0.7s ease;
```

Comment:
I've heard that `ms` for transitions is more performant. I can't remember when, or by whom, and it could be lies.